service_identity/src/main/kotlin/org/thechance/service_identity/domain/usecases/util/error_codes.kt

```diff
@@ -1,4 +1,4 @@
-package org.thechance.service_identity.endpoints.validation
+package org.thechance.service_identity.domain.usecases.util
```

Comment:
change file name please

Reply:
What would be the appropriate alteration?, How can the name be modified without changing its meaning?